### PR TITLE
Update KVS Schema and utilize a new feature

### PR DESCRIPTION
**Pull request by rejasupotaro**
# Overview

I updated KVS schema from `3.0.0` to `5.0.0`.

# New feature: Default value

Some features were introduced in this update. A change that affects to this project is [Support default value](https://github.com/rejasupotaro/kvs-schema/pull/18). If a field is difined as a constant, KVS schema reads and assigns the value to the generated class as a default value like below.

```java
// Schema class
@Table(name = "io.github.droidkaigi.confsched_preferences")
public abstract class DefaultPrefsSchema {
    @Key(name = "notification_setting")
    final boolean notificationFlag = true;
```

```java
// Generated class
  public boolean getNotificationFlag() {
    return getBoolean("notification_setting", true); // A constant value we set is specified.
  }
```

So we don't have to specify default values expressly.

```java
prefs.getNotificationFlag(); // It returns true if `notification_setting` is not stored.
prefs.getNotificationFlag(true); // Specifying default value is also fine.
```

### CODE REVIEW

**Comment by rejasupotaro** [comment]
@konifar Please review 🙏

**Comment by konifar** [comment]
👀 Thanks! I'll review it after going back home.

File: app/src/main/java/io/github/droidkaigi/confsched/prefs/DefaultPrefsSchema.java [hunk 1]
```diff
@@ -5,12 +5,12 @@
 
 @Table(name = "io.github.droidkaigi.confsched_preferences")
 public abstract class DefaultPrefsSchema {
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/prefs/DefaultPrefsSchema.java]: Can we change it to interface? If we can do that we don't have to make each variables final.

**Comment by hotchemi** [comment]
👀

File: app/src/main/java/io/github/droidkaigi/confsched/prefs/DefaultPrefsSchema.java (hunk 1 above)
Comment (rejasupotaro) [inline app/src/main/java/io/github/droidkaigi/confsched/prefs/DefaultPrefsSchema.java]: Yes, we can! I updated it in https://github.com/konifar/droidkaigi2016/pull/371/commits/40df785c23a42a3e6dd60be3340db082a52be7f9

Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/prefs/DefaultPrefsSchema.java]: Cool!

**Comment by konifar** [comment]
👀

**Comment by konifar** [comment]
LGTM
Thanks for great update! 🆒

**Pull request merged**